src/main/java/lotto/validation/BaseValidator.java

```diff
@@ -0,0 +1,22 @@
+package lotto.validation;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lotto.constatnt.ExceptionMessage;
+
+public abstract class BaseValidator {
+
+    protected void checkSize(List<Integer> numbers, int requiredCount) {
```

Comment:
개인적인 느낌이지만, `checkXXX()` 라는 메서드는 `boolean`을 반환해 줄 것 같네요.
예외를 터뜨린다면 조금 더 강하게 "검증"한다는 느낌의 단어를 사용해 주어도 좋을 것 같아요.
```suggestion
    protected void validateSize(List<Integer> numbers, int requiredCount) {
```

Reply:
말씀하신대로 메서드명을 좀 더 직관적이게 바꿀 필요가 있어 보이네요! 피드백 감사합니다😊